setup.cfg

```diff
@@ -26,7 +26,7 @@ packages =
 include_package_data = true
 
 install_requires =
-    numpy>=1.9.0
+    numpy>=1.19.5
```

Comment:
Is this the new lowest required verison for numpy.integer?

It's very high which means we'd get issues with some distributions that ship an older version, for example CentOS Stream 8 has 1.14.3 and CentOS Stream 9 has 1.20.1, I haven't checked Ubuntu.

Reply:
You are right, we do not need to change the lowest. The `Integer` member has always been there. However, the `int` is the one that was removed in the 1.24.0 version. I will amend my patch